### PR TITLE
adding domains & lowercase

### PR DESCRIPTION
**Pull request by florian-crtl**

### CODE REVIEW

File: list.txt [hunk 1]
```diff
@@ -12533,6 +12685,7 @@ gmail-box.com
 gmail-fiji.gq
 gmail.ax
 gmail.co5
+gmail.com
```
Comment (FGRibreau) [inline list.txt]: gmail.com is not a disposable email provider :)

Comment (florian-crtl) [reply list.txt]: Woops mb, i'll recheck and commit :)

Comment (florian-crtl) [reply list.txt]: Done

File: list.txt [hunk 2]
```diff
@@ -32213,6 +32641,7 @@ yahomail.gdn
 yahomail.top
 yahoo-mail.ga
 yahoo.co.au
+yahoo.com
```
Comment (FGRibreau) [inline list.txt]: I cannot accept your PR, to many changes and this is another example of error. Sorry, you broke my trust :s...

**Pull request closed without merging**